httpx/_api.py

```diff
@@ -114,6 +114,11 @@ def stream(
     cert: CertTypes = None,
     trust_env: bool = True,
 ) -> StreamContextManager:
+    """
+    Streaming response content.
```

Comment:
What do we think of some more elaborate copy like this...?

```
Alternative to `httpx.request()` that streams the response body instead of loading it into memory at once.

**Parameters**: See `httpx.request`.

See also: [Streaming Responses][0]

[0]: /quickstart#streaming-responses
```

Also I think we could add this docstring onto `Client.stream` and `AsyncClient.stream` as well. :-)

Reply:
Nice 👍 
But there are no `Client.stream` and `AsyncClient.stream` methods 😄